### PR TITLE
chore: remove unused deps

### LINKED ISSUE #1952: Daily build fails
Both `cargo udeps` and `cargo audit` fail in the [daily check](https://github.com/fedimint/fedimint/actions/runs/4433903728). I noticed it due to #1898 when trying to build all outputs of our flake.

TODO:
- [ ] remove unused deps #1962
- [x] update deps with vulnerabilities

### PR DESCRIPTION
**Pull request by GetPsyched**
Fixes  #1952

### CODE REVIEW

**Comment by elsirion** [comment]
`aquamarine` is a false-positive, we actually need it for building our docs. Can you leave it in and ignore in the check?

File: modules/fedimint-ln-client/Cargo.toml [hunk 1]
```diff
@@ -16,7 +16,6 @@ path = "src/lib.rs"
 anyhow = "1.0.66"
 async-trait = "0.1"
 async-stream = "0.3.5"
-aquamarine = "0.3.0"
```
Comment (dpc) [inline modules/fedimint-ln-client/Cargo.toml:19]:
```
[package.metadata.cargo-udeps.ignore]
# cargo udeps can't detect that one
normal = ["aquamarine"]
```

**Comment by elsirion** [comment]
As CI tells you, you'll also have to add the dependency back in ;)

**Comment by GetPsyched** [comment]
CI is just out for me at this point >:(

**Review by elsirion — APPROVED** [review]
Next time, please just amend commits and don't undo changes from earlier commits in later ones :)

**Comment by GetPsyched** [comment]
Sorry, I'll take care from now!

**Pull request merged**